website/docs/cloud-docs/api-docs/policy-checks.mdx

```diff
@@ -250,3 +250,150 @@ The GET endpoints above can optionally return related resources, if requested wi
+
+| Parameter                          | Description                                                                                                                                                                                                                                                                                                                 |
+|------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `page[number]`                     | **Optional.** If omitted, the endpoint will return the first page.                                                                                                                                                                                                                                                          |
+| `page[size]`                       | **Optional.** If omitted, the endpoint will return 20 agent pools per page.                                                                                                                                                                                                                                                 |
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request GET \
+  https://app.terraform.io/api/v2/task-stages/ts-rL5ZsuwfjqfPJcdi/policy-evaluations
+```
+
+### Sample Response
+
+```json
+{
+   "data":[
+      {
+         "id":"poleval-8Jj9Hfoz892D9WMX",
+         "type":"policy-evaluations",
+         "attributes":{
+            "status":"passed",
+            "policy-kind":"opa",
+            "status-timestamps":{
+               "passed-at":"2022-09-16T01:40:30+00:00",
+               "queued-at":"2022-09-16T01:40:04+00:00",
+               "running-at":"2022-09-16T01:40:08+00:00"
+            },
+            "created-at":"2022-09-16T01:39:07.782Z",
+            "updated-at":"2022-09-16T01:40:30.010Z"
+         },
+         "relationships":{
+            "policy-attachable":{
+               "data":{
+                  "id":"ts-yxskot8Gz5yHa38W",
+                  "type":"task-stages"
+               }
+            },
+            "policy-set-outcomes":{
+               "links":{
+                  "related":"/api/v2/policy-evaluations/poleval-8Jj9Hfoz892D9WMX/policy-set-outcomes"
+               }
+            }
+         },
+         "links":{
+            "self":"/api/v2/policy-evaluations/poleval-8Jj9Hfoz892D9WMX"
+         }
+      }
+   ]
+}
+```
+
+
+## List the Policy Outcomes of the Policy Evaluation
```

Comment:
hm, are we saying "policy check" or "policy evaluation"? I think I've seen both and I think it says "policy checks" in the UI (right? lmk if I'm wrong on that though!)

Reply:
![image](https://user-images.githubusercontent.com/17270065/191886929-d64cdb7c-ff60-4147-bead-f93613ec30f5.png)
For OPA policies, it does not say policy checks. We internally refer to it as evaluations.
Adding @hcrhall for his input here.
Happy to keep it consistent. Either ways, let me know your thoughts?  :)